common/app/views/emailFragment.scala.html

```diff
@@ -1,10 +1,26 @@
 @import com.gu.identity.model.EmailEmbed
 @(page: model.Page, emailType: String, listName: String, emailEmbedData: EmailEmbed)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-@emailEmbed(page) {
-    @fragments.email.signup.subscription.emailSignUp(
-        emailType,
-        listName,
-        emailEmbedData
+@electionNewsletters = @{
+    List(
+        "minute-us", "us-morning-newsletter", "today-us", "green-light", "business-today",
+        "the-recap", "the-fiver", "the-spin", "the-breakdown", "sports-au"
     )
 }
+
+@emailEmbed(page) {
+    @emailEmbed(page) {
```

Comment:
Is this there twice on purpose?

Reply:
Ah good catch. Nope. It's a copy and paste mistake that IntelliJ fixed the brackets on, meaning I totally missed it.